rust/cubestore/cubestore/Cargo.toml

```diff
@@ -34,10 +34,10 @@ bytes = "0.5.4"
 serde_json = "1.0.56"
 futures = "0.3.5"
 smallvec = "1.8.0"
-flexbuffers = "0.1.1"
+flexbuffers = "0.2.2"
 byteorder = "1.3.4"
 log = "=0.4.11"
-simple_logger = "1.7.0"
```

Comment:
Same as previous. `Cargo.lock` should be updated as well.

Reply:
as pew flexbuffer, an error is thrown if I use the original simple_logger, around missing default time zone

Reply:
```
> cargo run
2023-01-24 09:54:36,143 INFO [cubestored] <pid:3560> Cube Store version 0.31.48
thread 'main' panicked at 'Could not determine the UTC offset on this system. Possible causes are that the time crate does not implement "local_offset_at" on your system, or that you are running in a multi-threaded environment and the time crate is returning "None" from "local_offset_at" to avoid unsafe behaviour. See the time crate's documentation for more information. (https://time-rs.github.io/internal-api/time/index.html#feature-flags): IndeterminateOffset', /Users/jonas/.cargo/registry/src/github.com-1ecc6299db9ec823/simple_logger-1.16.0/src/lib.rs:409:85
```